flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/RestHandlerConfiguration.java

```diff
@@ -38,18 +38,23 @@
 
 	private final File webUiDir;
 
+	private final long metricFetcherUpdateInterval;
```

Comment:
this doesn't _really_ belong into the handler configuration imo, but we don't have a better place for it do we :/